src/storage/ducklake_insert.cpp

```diff
@@ -621,7 +621,12 @@ PhysicalOperator &DuckLakeInsert::PlanCopyForInsert(ClientContext &context, Phys
 	physical_copy.names = std::move(copy_options.names);
 	physical_copy.expected_types = std::move(copy_options.expected_types);
 	physical_copy.parallel = true;
-	physical_copy.hive_file_pattern = !is_encrypted ? true : false;
+	if (is_encrypted) {
+		// if encrypted we never use hive partitioning
+		physical_copy.hive_file_pattern = false;
```

Comment:
I think for encrypted databases we just want to default this setting to `false` on creation - there might be cases where people want to write encrypted databases in a hive file pattern. The setting should still be respected.